### ISSUE TITLE
There is no way to get all tweet replies.

### ISSUE THREAD

**Issue by 1guzzy** [PLhery/node-twitter-api-v2#252]
**Is your feature request related to a problem? Please describe.**
From my understanding, there is no way to get all the replies of a tweet. It seems like the only way to get the replies of a tweet is to hit the `/search/recent` route and supply a query param `conversation_id` which is the tweet id.

**Describe the solution you'd like**
A function that returns all tweet replies.

**Describe alternatives you've considered**
It might be possible to use the v2 `search` function to solve this. The issue with this function is that it requires you supply it some string to search for, when in this case all we need to supply is the query param `conversation_id`

**Additional context**
N/A

Sorry if there is already a way to do this and I've missed it!

**Comment by alkihis** [comment]
Hi,
Yeah an implementation with object-only parameter can be made for `.v2.search`. You can still give an empty string, it should work too.

**Comment by 1guzzy** [comment]
Just double checked, empty string throws this error.

```
node_modules/twitter-api-v2/dist/client-mixins/request-handler.helper.js:99
        return new types_1.ApiResponseError(errorString, {
               ^
Error: Request failed with code 400 - Invalid Request: One or more parameters to your request was invalid
```

**Comment by alkihis** [comment]
Available on 1.11.3 :)
